mne/minimum_norm/inverse.py

```diff
@@ -769,9 +769,7 @@ def _check_loose_forward(loose, forward):
 
 def _check_reference(inst, ch_names=None):
     """Check for EEG ref."""
-    info = inst.info.copy()  # make a copy as comps are now hacked:
-    # XXX : ugly hack to avoid picking subset of info with applied comps
-    info['comps'] = []
+    info = inst.info
```

Comment:
should this be a copy?

Reply:
I don't think this needs to be a copy.
I just reverted to the code prior to #5388

Furthermore the channel selection is done in 
https://github.com/mne-tools/mne-python/blob/349636572155eced005e2105962467eabaa63261/mne/minimum_norm/inverse.py#L778
which returns a copy by default.

and `_needs_eeg_average_ref_proj` doesn't modify `info`